### PR TITLE
comit to feature1

### PR DESCRIPTION
**Pull request by okorol90**
merge there

### CODE REVIEW

File: file1.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-Hello World!!! -Hi
\ No newline at end of file
+Hello World!!! -Hi
+
```
Comment (okorol90) [inline file1.txt:2]: change it

File: file1.txt [hunk 2]
```diff
@@ -1 +1,3 @@
...
+I'm in orign
```
Comment (annasemkaobg) [inline file1.txt:3]: need to be fix

**Review by okorol90 — COMMENTED** [review]
sdfgs

**Review by annasemkaobg — COMMENTED** [review]
save

**Comment by okorol90** [comment]
xfddf

**Pull request merged**